### ISSUE TITLE
V-73387 Hardcodes Domain to "testdomain.com"

### ISSUE THREAD

**Issue by slipthebit** [mitre/microsoft-windows-server-2016-stig-baseline#40]
The `dsquery` command in V-73387.rb has the domain hardcoded to "testdomain.com".
The control should use `forrest` as defined in inspec.yml.
```
 ...
   forrest = attribute('forrest')
   domain_role = command('wmic computersystem get domainrole | Findstr /v DomainRole').stdout.strip
   if domain_role == '4' || domain_role == '5'
     query = command("dsquery * \"cn=Default Query Policy,cn=Query Policies,cn=Directory Service, cn=Windows NT,cn=Services,cn=Configuration,dc=testdomain,dc=com\" -attr LDAPAdminLimits").stdout
   ldap_admin_limits = parse_config(query.gsub(/;/, "\n")).params
...
```

**Comment by aaronlippold** [comment]
You are correct - we should be using the input.
--------
Aaron Lippold

***@***.***

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

**Comment by slipthebit** [comment]
Closed initial PR to clean up branching. Opened PR #45 with isolated code change.

**Comment by karikarshivani** [comment]
Duplicate of #43 and addressed by PR #45
